packages/io/src/adapter.ts

```diff
@@ -15,10 +24,9 @@ export class KoaRouterAdapter extends KoaRouter {
       ctx.body = await this.cor.dispatch({
         method: ctx.method,
         path: ctx.path,
-        params: ctx.params,
-        // todo: support body & query
-        // query: ctx.query,
-        // body: ctx.request.body,
+        query: ctx.query as JsonValue,
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        body: ctx.request.body || undefined,
```

Comment:
Is falsy value disallowed to be assigned on `body`, so add `undefined` as a fallback?

Reply:
No, I just thought there would be no body when get method, so I added an undefined, but after thinking about it, I don't really need it.